app/scripts/views/base.js

```diff
@@ -151,8 +151,11 @@ define(function (require, exports, module) {
       NotifierMixin.initialize.call(this, options);
 
       // Prevent errors from being displayed by aborted XHR requests.
-      this._boundDisableErrors = _.bind(this.disableErrors, this);
+      this._boundDisableErrors = this.disableErrors.bind(this);
```

Comment:
> Or (even better) have we dropped support for oldIE?

We have basically said "meh" to anything below IE9. IE9 supports bind. There are older Android browsers that do not have Function.prototype.bind, and we should probably worry about those users opening the verification links and breaking.

So yes, we should polyfill to support verification on old Android browsers, but that's not a super high priority. See https://github.com/mozilla/fxa-content-server/issues/2886.